demo.java

```diff
@@ -0,0 +1,8 @@
+import java.util.*;
+public class demo
```

Comment:
Better practice to create class names start letter in upper case, like public class **Demo** instead of public class **demo**